marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-card.tsx

```diff
@@ -258,6 +258,9 @@ const MappingCard: React.FC<Props> = (props) => {
 
     const updateMappingWithNamespaces = async (mapDataLocal) => {
         let {lastUpdated, ...dataPayload} = mapDataLocal;
+        if(nmspaces.hasOwnProperty("")){
```

Comment:
rather than deleting an invalid namespace property, is it possible to not add/create such namespace in the first place.

Reply:
Done.